### ISSUE TITLE
🐛 BUG: "You must supply options.input to rollup"

### ISSUE THREAD

**Issue by ManzDev** [snowpackjs/astro#1561]
### What package manager are you using?

npm 8.0.0

### What operating system are you using?

Windows 10 + WSL2

### Describe the Bug

When run `npm run build`, snowpack show this message:

```bash
[11:53:24] [snowpack] Ready!
[build] ! building pages...
[build] ✔ pages built.
[build] ! optimizing css...
[build] ✔ css optimized.
[build] ! creating sitemap...
[build] ✔ sitemap built.
[build] ! copying public folder...
[build] ✔ public folder copied.
[build] ! bundling...
[build] You must supply options.input to rollup
```

The last message is a error. All works okay but.... Why this error message? I understand that rollup layer need a option.input value, but... How fix this from Astro?

My Astro config ( astro.config.mjs ):

```js
export default ({
  projectRoot: ".",
  dist: "./dist",
  src: "./src",
  pages: "./src/pages",
  public: "./public",
  renderers: [],
  buildOptions: {
    site: "https://DOMAINURL/",
    sitemap: true,
    pageUrlFormat: "directory"
  },
  devOptions: {
    hostname: "localhost",
    port: 3000,
    trailingSlash: "always"
  }
});
```

My Snowpack config ( snowpack.config.mjs ):

```js
export default {
  alias: {
    $components: "./src/components",
    $modules: "./src/modules",
    $public: "./public"
  }
}
```

### Steps to Reproduce

1. `npm run build`

**Comment by ManzDev** [comment]
Great! I like chain `npm run build && npm run deploy`, but `npm run build` return a error code and `npm run deploy` fail.
I will wait for 0.21!

**Comment by oliverturner** [comment]
Is there a way to suppress the error? It breaks Netlify deploys...

I've tried passing a value in via `snowpack.config.mjs`
```
packageOptions: {
    rollup: {
        input: "./lib/noop.js",
    },
},
```
...but without success

**Comment by natemoo-re** [comment]
I'm going to assume this is fixed by v0.21, so we'll track it against that milestone. If this is still an issue in `astro@next` please let us know!

**Comment by natemoo-re** [comment]
Following up on my previous comment, the team has decided to close out issues that have been confirmed as fixed by [astro@0.21.0-next.0](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.0), [astro@0.21.0-next.1](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.1), or [astro@0.21.0-next.2](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.2). Our hope is that this will help the [v0.21 milestone](https://github.com/snowpackjs/astro/milestone/1) remain as actionable as possible.

To verify that this issue has been fixed, you may
- Open a new [`astro@next` project on StackBlitz](https://astro.new/minimal@next)
- Locally upgrade to the `astro@next` preview
    ```shell
    npm i astro@next
    # If using Framework renderers...
    npm i @astrojs/renderer-name@next
    ```

If you think we made a mistake, **please reopen this issue**!

If you run into any other problems with `astro@next`, please [open a new issue](https://github.com/snowpackjs/astro/issues/new?assignees=&labels=0.21&template=---01-bug-report.yml&title=%F0%9F%90%9B+BUG%3Aastro@next).
